### PR TITLE
Circuitverse UI update

### LINKED ISSUE #3381: Forum UI update
**Is your feature request related to a problem? Please describe.**
 Forum UI update. It's an ndividual issue regarding this issue #3316 

**Describe the solution you'd like**
I will first of all create an updated UI in Figma as an prototype and then will customize the Forum UI

**Are you working on this? (Yes/No)**
Yes

**Tasks**

- [x] Design Prototype [Click to open design](https://www.figma.com/file/I6qAJbSV4W69NNWZbNdcr7/Circuitverse-Forum-UI?node-id=0%3A1&t=H0tSjwun90CBL96d-1)
- [x] Fix all the broken links and errors
- [x] Update UI

### PR DESCRIPTION
**Pull request by Tanmoy741127**
Fixes #3381 

#### Note -
Created an PR https://github.com/CircuitVerse/simple_discussion/pull/1 in the repo https://github.com/CircuitVerse/simple_discussion.
*Need to first merge that pull request , then we can proceed for merging this pull request*

#### Describe the changes you have made in this PR -
- Upgrade to latest version of simple_discussion from https://github.com/CircuitVerse/simple_discussion
- Override themes that is necessary

### Screenshots of the changes (If any) -
| Page Name | Screenshot |
|-----------|--------------|
| All Threads | image |
| Thread Details | image |
| Post Edit | image |
| New Question | image |
| Login Bar | image | 
| My Questions | image |
| Participating | image |
| Answered | image |
| Unanswered | image | 

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by Tanmoy741127** [comment]
@tachyons 
as per yesterday's dicussion ,
I have updated the https://github.com/CircuitVerse/simple_discussion repo and created an PR regarding that https://github.com/CircuitVerse/simple_discussion/pull/1 .

I have also updated the README of simple_discussion to guide about theming for color updates.

In CircuitVerse repo, in this pull request, I have updated the gemfile with the github link of simple_discussion and do the necessary changes for theming.

**Comment by tachyons** [comment]
CI is failing since you forgot to commit Gemfile.lock

**Comment by Tanmoy741127** [comment]
OK

**Comment by Tanmoy741127** [comment]
> CI is failing since you forgot to commit Gemfile.lock

Ok ! Updated in latest commit.

File: app/assets/stylesheets/simple_discussion_overrides.scss [hunk 1]
```diff
@@ -2,3 +2,4 @@
   margin: 0 auto;
   width: 95%;
 }
+
```
Comment (tachyons) [inline app/assets/stylesheets/simple_discussion_overrides.scss:5]:
```suggestion
```

**Pull request merged**